python/ray/serve/tests/test_autoscaling_policy.py

```diff
@@ -357,6 +360,42 @@ def test_smoothing_factor_with_0_replica():
     assert new_num_replicas == 1
 
 
+def test_smoothing_factor_scale_down_to_0_replica():
+    """Test that a deployment scales down to 0 for non-default smoothing factors."""
+
+    # Smoothing factor = 10
+    config = AutoscalingConfig(
+        min_replicas=0,
+        max_replicas=5,
+        smoothing_factor=10,
+        upscale_delay_s=0,
+        downscale_delay_s=0,
+    )
+    policy = BasicAutoscalingPolicy(config)
+    new_num_replicas = policy.get_decision_num_replicas(
+        current_num_ongoing_requests=[0, 0, 0, 0, 0],
+        curr_target_num_replicas=5,
+        current_handle_queued_queries=0,
+    )
+
+    # 1 * 10
```

Comment:
What does this mean?

Reply:
My bad, that was copy pasted - I've removed the comment